assignments/arrays.js

```diff
@@ -63,32 +313,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+const arr = inventory
+for (let i = 0; i < arr.length; i++) {
+    if (arr[i].id === 33) {
+        console.log(`Car 33 is a ${arr[i].car_year} ${arr[i].car_make} ${arr[i].car_model}`);
+    }
+}
+//console.log(`Car 33 is a ${arr[i].car_year} ${arr[i].car_year} *car model goes here*`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+
 let lastCar = 0;
-console.log();
+lastCar = arr[arr.length - 1]
+console.log(`last car is a ${lastCar.car_make} ${lastCar.car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+
+let carModels = arr;
+carModels.sort(function (a, b) {
```

Comment:
I made your life more complicated than it needed to be. We only needed to just get the strings and sort that. 😅 Sorry!